dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/statelisteners/PopulateEntryListener.java

```diff
@@ -0,0 +1,55 @@
+/*
+ *    Copyright 2019 OICR
```

Comment:
Probably should move to 2021 and add UCSC (should be a few examples of this in the codebase)